addons/compat_rhs_afrf3/compat_rhs_afrf3_repair/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,5 @@
+class Extended_PreInit_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_SCRIPT(XEH_preInit));
```

Comment:
path macros and `addon` won't work because it's a subaddon
I think the easiest fix is to just keep the code behind the `isModLoaded` check in upper

Reply:
Reverted.